### ISSUE TITLE
Lots of print output in 0.11

### ISSUE THREAD

**Issue by mands** [EmilStenstrom/django-components#63]
Hi,

Thanks for the useful library! I've recently upgraded to 0.11 and have noticed a lot of print output in my console from the djagno server when viewing pages that have components on them, as shown below - reverting to 0.10 appears to fix the issue.

```html
------------------------------------ Before ------------------------------------
<django.template.defaulttags.LoadNode object at 0x7f2af5438640>
<TextNode: '\n\n<div>\n  <div class="rel'>
<Variable Node: columns>
<TextNode: '">\n    '>
<ForNode: for object in object_list, tail_len: 3>
<TextNode: '\n    </div>\n</div>\n  '>
<IfNode>
	<TextNode: '\n    <div class="w-full m'>
	<Component Node: <dp.apps.dp_core.components.components.Pagination object at 0x7f2af543fa60>. Contents: None>
	<TextNode: '\n      </div>\n\n  '>
<TextNode: '\n</div>\n'>
------------------------------------ After -------------------------------------
<django.template.defaulttags.LoadNode object at 0x7f2af5438640>
<TextNode: '\n\n<div>\n  <div class="rel'>
<Variable Node: columns>
<TextNode: '">\n    '>
<ForNode: for object in object_list, tail_len: 3>
<TextNode: '\n    </div>\n</div>\n  '>
<IfNode>
	<TextNode: '\n    <div class="w-full m'>
	<Component Node: <dp.apps.dp_core.components.components.Pagination object at 0x7f2af543fa60>. Contents: None>
	<TextNode: '\n      </div>\n\n  '>
<TextNode: '\n</div>\n'>
```

**Comment by EmilStenstrom** [comment]
@mands The problem was that I has local changes to my repo when I built the 0.11 release, and those changes propagated to the release, not the latest master. **It's now fixed, as version 0.12 is released**. I also yanked (removed) the 0.11 version as I don't want to fill people's logs with junk.

Thank you so much for reporting.

**Comment by mands** [comment]
Great - that's for the quick turnaround!
